runners/julia-dockerfile

```diff
@@ -7,6 +7,9 @@ ENV JULIA_DEPOT_PATH="/julia_depot/"
 # Pre-compiling Julia dependencies
 RUN  julia -e 'pwd(); using Pkg; \
     Pkg.add.(["SpeciesDistributionToolkit", "Dates", "Clustering", "JSON", "CSV", "DataFrames", "StatsBase", "EvoTrees", "MultivariateStats" ]); \
-    Pkg.instantiate();'
+    Pkg.instantiate(); Pkg.precompile()'
```

Comment:
Do we want to keep the precompile step since every pipeline is running in its own environment?

Reply:
long story shart: everything is precompiled in `/julia_depot` for now, and the main script here activates `/julia_depot` as the project dir